File: Daily Flash/week 4/day 1/18-DailyFlash_Solutions/3_Feb_Solutions_Two/C/program4.c

#include <stdio.h>
#include <stdlib.h>

int main () {

	int n, k = 0;
	printf("Enter Size : ");
	scanf("%d", &n);

	if(n < 0)
		exit(0);
	for(int i = 1; i <= n; i++){
		for(int j = 0; j <= (n - i)*2; j=j+2){
			printf("%c ", 64 + i + j );
		}
		printf("\n");
	}
}

File: Daily Flash/week 4/day 1/18-DailyFlash_Solutions/3_Feb_Solutions_Two/Java/program4.java

import java.io.*;

class Program {
	public static void main(String[] args) throws IOException{
			
		int n = 0, flag = 0;
		System.out.printf("Enter Size : ");
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

		try{
			n = Integer.parseInt(br.readLine());
		}
		catch(NumberFormatException ne){
			System.exit(0);
		}

		if(n < 0)
			System.exit(0);
		for(int i = 1; i <= n; i++){
			for(int j = 0; j <= (n - i)*2; j=j+2){
				System.out.printf("%c ", 64 + i + j );
			}
			System.out.printf("\n");
		}
	}
}